prov/efa/src/rxr/efa_tp_def.h

```diff
@@ -1,3 +1,7 @@
+#include <config.h>
+
+#if INCLUDE_LTTNG
```

Comment:
This patch looks like it should have been merged with the previous patch.  Don't patch a patch that hasn't been merged.  Just update the original patch directly.

Reply:
I need clean up patches around. Wei commented some changes will cause additional changes around. I will get them in one by one.